tests/requirements/requirements_base.txt

```diff
@@ -12,4 +16,4 @@ mock
 # Unreleased django-cms 4.0 compatible packages
 https://github.com/django-cms/django-cms/tarball/release/4.0.1.x#egg=django-cms
 https://github.com/django-cms/djangocms-versioning/tarball/support/django-cms-4.0.x#egg=djangocms-versioning
-https://github.com/Aiky30/djangocms-moderation/tarball/master#egg=djangocms-moderation
+https://github.com/FidelityInternational/djangocms-moderation/tarball/feature/django-42-compat#egg=djangocms-moderation
```

Comment:
@vipulnarang95 , 
please use `https://github.com/django-cms/djangocms-moderation/tarball/master#egg=djangocms-moderation` , as **Fabian** has already finished Django 4.2 support on it and merged in master branch.

Reply:
done